src/api/telegram.ts

```diff
@@ -0,0 +1,16 @@
+import { TOKEN, CHAT_ID } from "../values";
```

Comment:
where is _values_ file?

Reply:
I added it to gitignore

Reply:
I couldn't manage with an error that appeared when I used .env and I created value.js for now